web/src/main/java/se/inera/intyg/webcert/web/web/controller/testability/facade/dto/CertificatePatientsResponseDTO.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2021 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.webcert.web.web.controller.testability.facade.dto;
+
+import java.util.List;
+import se.inera.intyg.common.support.facade.model.Patient;
+
+public class CertificatePatientsResponseDTO {
```

Comment:
Why don't you use lombok annotation for data and constructor?

Reply:
I would like to :-) But currently we are using AutoValue in Webcert-web and as I understands it, it is not possible to use both inside the same "module".